### DISCUSSION TITLE
Add a "Notify Me" option for the frontend of WordPress

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1284] [Feature Requests]
This is potentially a very useful feature:

> Can you please tell me if it's possible to let logged-in members subscribe by email to specific categories or authors (through the front end)?

**Comment by andergmartins** [comment]
Cool, I agree. This could be a widget, added to the content, and maybe a shortcode as well?
In addition to that, each Notification workflow could optionally provide a shortcode that prints a "notify me" button (accepting custom label and description).

**Comment by stevejburge** [comment]
One more request for this: "We are building a platform where users have to "subscribe" on their most favourite posts."

**Comment by stevejburge** [comment]
One more request:

> Is there function where I can turn on the button where user can "subscribe" to get notifications on changes in blogs ?

**Comment by stevejburge** [comment]
One more request:

> does it have a subscribe to the blog feature?

**Comment by stevejburge** [comment]
One more request:

> is there a way i can add maybe a checkbox or something on the frontend of the site? So if a user logs in they can see "get content notifications" checkbox and tick or untick depending on if they want email notifications?

**Comment by stevejburge** [comment]
One more:

> I am trying to make this possible:
> 
> Automatically send a notification to specific users / roles when a post is published in a specific category
> 
> In other words, if a post is published in the category "News", a notification will be sent automatically to all users / roles that are linked/subscribed to the NEWS category.
> 
> Or if a post is published in the category "TRENDY", a notification will also be sent out here to the users / roles, which is linked/subscribed to the TRENDY category.

**Comment by rizaardiyanto1412** [comment]
One more:

> Dear Support team, is it possible to use the "Notify me" option in PublishPress so a logged in user can optin somehow on the frontend to get notified when the post he opted in for is changed
